deploy/e2e/apply/es6-master.yaml

```diff
@@ -1,24 +1,23 @@
 apiVersion: apps/v1
-kind: Deployment
+kind: StatefulSet
 metadata:
-  name: es-master
+  name: es6-master
 spec:
-  minReadySeconds: 60
-  strategy:
-    type: RollingUpdate
-    rollingUpdate:
-      maxSurge: 0
-      maxUnavailable: 1
+  replicas: 1
+  serviceName: es6-master
   selector:
     matchLabels:
-      application: es-master
-  replicas: 1
+      application: es6
+      role: master
   template:
     metadata:
       labels:
-        application: es-master
+        application: es6
+        role: master
     spec:
       serviceAccountName: operator
+      securityContext:
+        fsGroup: 1000
```

Comment:
What's the purpose of setting fsGroup?

Reply:
took this from the Elasticsearch helm chart. Not sure if it's needed, but it also shouldn't hurt. https://github.com/elastic/helm-charts/blob/master/elasticsearch/templates/statefulset.yaml#L49

Reply:
Don't know why they put "runAsUser: 0" and "privileged: true" btw. https://github.com/elastic/helm-charts/blob/master/elasticsearch/templates/statefulset.yaml#L110

Reply:
What about I add fsGroup: 1000 consistently in all the manifests?